html/app.js

```diff
@@ -106,6 +107,7 @@ $(".request-identity-button").click(function(e){
 $(document).on("click", ".job-page-block", function(e){
     e.preventDefault();
     selectedJobId = $(this).data('job');
+//    selectedJobId["application"] = $(this).data('application')
```

Comment:
Do we need this?

Reply:
the variable application does not exist in the file.